README.md

```diff
@@ -126,16 +126,59 @@ The trace agent can be configured by passing a configurations object to the agen
 
 One configuration option of note is `enhancedDatabaseReporting`. Setting this option to `true` will cause database operations for redis and MongoDB to record query summaries and results as labels on reported trace spans.
 
-## Disabling the trace agent
+### Disabling the trace agent
 
 The trace agent can be turned off by either setting the `GCLOUD_TRACE_DISABLE` environment variable or specifying `enabled: false` in your configuration file.
 
-## Trace batching and sampling
+### Trace batching and sampling
 
 The aggregation of trace spans before publishing can be configured using the `flushDelaySeconds` and `bufferSize` [options](config.js). The spans recorded for each incoming requests are placed in a buffer after the request has completed. Spans will be published to the UI in batch when the spans from `bufferSize` requests have been queued in the buffer or after `flushDelaySeconds` have passed since the last publish, whichever comes first.
 
 The trace configuration additionally exposes the `samplingRate` option which sets an upper bound on the number of traced requests captured per second. Some Google Cloud environments may override this sampling policy.
 
+### Tracing Additional Modules
+
+In addition to the modules listed [above](#what-gets-traced), the trace agent can be configured to trace additional modules through the use of *plugins*. To load an additional plugin, specify it in the agent's configuration:
+
+```javascript
+  require('@google/cloud-trace').start({
+    plugins: {
+      // You may use a package name or absolute path to the file.
+      'my-module': '@google/cloud-trace-plugin-my-module',
+      'another-module': path.join(__dirname, 'path/to/another/module/plugin.js')
```

Comment:
nit: the example suggest strong coupling between `another-module` and the where the plugin file comes from. That is, makes me think that `plugin.js` is part of `another-module` that I must dig out. I suggest:

`path.join(__dirname, 'path/to/my-custom-plugins/plugin-another-module.js')`